scalableminds/webknossos
# When re-activating user, check orga user limit

### Steps to test:
- for an orga, register several users
- in db, set user limit
- deactivate some users so that the orga user count is within limits, then try to re-activate one, should be blocked
- after relaxing the limit in the db, it should be allowed again

### Issues:
- fixes #6757

------
